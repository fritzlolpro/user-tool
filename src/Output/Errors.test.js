import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ErrorText from './Errors';

describe('ErrorText', () => {

    it('should be defined', () => {
        expect(ErrorText).toBeDefined();
    });

    it('renders without crashing'),
    () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <ErrorText>test error msg</ErrorText>, div);
    }

    test('has a valid snapshot', () => {
        const component = renderer.create(
            <ErrorText>test error msg</ErrorText>
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})
