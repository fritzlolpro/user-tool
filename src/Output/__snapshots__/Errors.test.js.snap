// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorText has a valid snapshot 1`] = `null`;
